README.md

```diff
@@ -35,7 +35,7 @@ Add the following lines to your `Cargo.toml` file:
 
 ```toml
 [dependencies]
-rust-proof = "https://github.com/Rust-Proof/rustproof"
+rust-proof = "https://github.com/Rust-Proof/rustproof.git"
```

Comment:
I think this might actually have to be `rustproof = { git = "https://github.com/Rust-Proof/rustproof.git" }`